package.json

```diff
@@ -50,10 +50,10 @@
     "js-cookie": "^2.1.4",
     "jsdom": "^11.6.2",
     "jwt-decode": "2.2.0",
-    "lodash": "^4.17.11",
-    "medium-draft": "beta",
+    "lodash-es": "^4.17.11",
+    "medium-draft": "^0.6.0-beta1",
```

Comment:
Is this new beta1 `editor` flawless 👍 hehe

Reply:
Hahahah, yeee. I checked for an update, but we were running the latest version :/ (as you can see in yarn.lock)